### PR TITLE
Add more awesome links and collab code subject

### PR DESCRIPTION
**Pull request by juanmnl**
Added more links and Collaborative Live Code as a Subject.
Don't know your policy about blogs, but added two for js as they are really awesome and have learnt a lot from them.
I'll try to make a spanish version of this in the next days.
Thanks for starting this!

:)

### CODE REVIEW

**Comment by juanmnl** [comment]
I think there should be some labels too:  
- [free]
- [paid]
- [book]
- [blog]
- [tutorial]

**Comment by auchenberg** [comment]
Instead of generic blogs and sources, I prefer the list to contain specific articles, just like when you get a list of articles to read in school. Hence the point of this being a **curated** list :)

**Comment by juanmnl** [comment]
Good to know, even if I don't think those blogs are at all _generic_, but I get your point.  
Consider that linking specific articles could lead to quickly out-dated stuff and/or a gigantic list. That's the main problem of many **curated** repos I've seen.

**Comment by juanmnl** [comment]
Removed the blog links to the PR.

**Comment by auchenberg** [comment]
Well, that's the thing. I wanna capture what **we** consider the current "basic knowledge", not what's most hip/popular right now. There's already many lists of blogs/tweeps to follow, and this shouldn't be yet another one.

A link to MDN or JSBIN doesn't add much, unless we consider it the **go-to** tool that ever student needs to learn/use. It's a tough one to get right, but let's give it a try.

File: README.md [hunk 1]
```diff
@@ -74,7 +80,15 @@ A crowdsourced and curated list of material that's considered mandatory curricul
 * [Show Me The Numbers](http://www.amazon.com/Show-Me-Numbers-Designing-Enlighten/dp/0970601972/)
 
 ## Courses/classes
+* [Codecademy](https://www.codecademy.com/)
```
Comment (auchenberg) [inline README.md:83]: Can you link to specific courses from Codecademy you'd consider mandatory?

File: README.md [hunk 2]
```diff
@@ -74,7 +80,15 @@ A crowdsourced and curated list of material that's considered mandatory curricul
...
 * [Udacity Front-End Web Developer Nanodegree](https://www.udacity.com/course/front-end-web-developer-nanodegree--nd001)
 * [Front-end masters](https://frontendmasters.com/)
+* [Egghead.io](https://egghead.io/)
```
Comment (auchenberg) [inline README.md:86]: Be specific.

File: README.md [hunk 3]
```diff
@@ -74,7 +80,15 @@ A crowdsourced and curated list of material that's considered mandatory curricul
...
+* [Evented Mind](https://www.eventedmind.com/classes)
+
+## Collaborative Live Code
+* [JS Bin](http://jsbin.com/)
+* [JS Fiddle](https://jsfiddle.net/)
+* [Code Pen](http://codepen.io/)
```
Comment (auchenberg) [inline README.md:92]: Which one of them to recommend? Do they serve a different purpose?

File: README.md [hunk 4]
```diff
@@ -28,9 +29,11 @@ A crowdsourced and curated list of material that's considered mandatory curricul
 
 ## CSS
 * [MDN: CSS](https://developer.mozilla.org/en-US/docs/Web/CSS)
+* [Lea Verou: CSS Secrets](http://shop.oreilly.com/product/0636920031123.do)
 
 ## HTML
 * [MDN: HTML](https://developer.mozilla.org/en-US/docs/Web/HTML)
+* [HTML5 Rocks](http://www.html5rocks.com/)
```
Comment (auchenberg) [inline README.md:36]: A specific HTML5Rocks article in mind?

**Comment by juanmnl** [comment]
Oh ok. I'll try to be more specific about it then. I personally consider some of these blogs, websites, etc, as a dictionary. You don't tell students to look just at one word and that's it, it's the knowledge of its existence that's useful. Like StackOverflow, for example. You may not care about most of the "posts", but knowing what it is and that you can search there for answers is what's useful.  
Anyways, the best of lucks with this, i'll try to separate every useful thing from the links i proposed and submit another pr in the future.

:)

File: README.md (hunk 3 above)
Comment (juanmnl) [inline README.md:92]: To be fair, your front-end masters link didn't have any specifics at this time, that's why i added similar stuff.

Comment (auchenberg) [inline README.md:92]: Yeah I know. I just think we have to be specific, as we otherwise will end up creating yet another front-end list.

Comment (juanmnl) [inline README.md:92]: Yes! :) Agree 100% I already have a couple of more specific articles that could be added. I'll add a couple more tomorrow and push it so you can have a look.

**Pull request closed without merging**